src/main/java/core/basesyntax/MainApp.java

```diff
@@ -9,12 +9,12 @@ public static void main(String[] args) {
 
         Machine[] machinesArr = new Machine[]{truck,bulldozer,excavator};
 
-        for (Machine machine/element : machinesArr) {
-            machineStart.doWork();
+        for (int i = 0; i < machinesArr.length; i++) {
```

Comment:
FYI: 
you can use for each loop, just with the correct naming 
for example: 
for (Machine machine : machines) {
    machine.startWork();
}

Reply:
Oh, I see! It was my 1st variant =) And I changed it before the first PR =))